activiti-cloud-starter-runtime-bundle/src/test/resources/access-control.properties

```diff
@@ -1,4 +1,23 @@
-activiti.cloud.security.user.testuser.test-app.policy.read=process_pool1,ProcessWithVariables
-activiti.cloud.security.user.testadmin.test-app.policy.read=process_pool1,ProcessWithVariables
-activiti.cloud.security.user.hruser.test-app.policy.write=process_pool1,ProcessWithVariables,SimpleProcess,ProcessWithVariables2,ProcessWithBoundarySignal
-activiti.cloud.security.user.hradmin.test-app.policy.write=process_pool1,ProcessWithVariables,SimpleProcess,ProcessWithVariables2,ProcessWithBoundarySignal
\ No newline at end of file
+activiti.security.policies[0].name=MyPolicy for TestUser
```

Comment:
I like the new way of describing policies. It's easier to read.

Reply:
yeah! there are a lot of things to improve.. but this new approach simplified a lot of parsing code